tests/hpc/hpc_master.pm

```diff
@@ -0,0 +1,99 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: HPC master
+#    This test is aiming at comprehensive HPC testing. That means
+#    that most components of the HPC product should be installed and
+#    configured. At the same time it is expected that the HPC cluster
+#    set-up is fairly complex, so that NFS shares are mounted, required
+#    database(s) are ready to be used, some data is being populated to
+#    the databases, artificial users are added etc.
+# Maintainer: Sebastian Chlad <sebastian.chlad@suse.com>
+
+use base "hpcbase";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+
+sub mount_nfs {
+    ## TODO: move to the lib
+    systemctl 'start nfs';
+    systemctl 'start rpcbind';
+    record_info('show mounts aviable on the supportserver', script_output('showmount -e 10.0.2.1'));
+    assert_script_run("mkdir -p /shared/slurm");
```

Comment:
You could use single quotes when you don't want to do variable expansion, which may lead to slightly better performance.